packages/tc-schema-sdk/data-accessors/relationship-type.js

```diff
@@ -60,6 +62,17 @@ const formatSimpleRelationship = (
 	};
 };
 
+const formatRichRelationship = richRelationshipDefinition => {
+	const { from } = richRelationshipDefinition;
+	const { to } = richRelationshipDefinition;
+	const enriched = {
+		...richRelationshipDefinition,
+		from: {...from, otherNodeName: from.type },
+		to: {...to, otherNodeName: to.type },
```

Comment:
```suggestion
		from: {otherNodeName: from.type, ...from },
		to: {otherNodeName: to.type, ...to },
```

Assuming you want to default to whatever to/from provide (if defined) this is the patten to use. Properties get added from left to right, so this means the value you set here will get overridden by whatever already existed on to/from